ignite/pkg/cosmosbuf/buf.go

```diff
@@ -167,9 +172,35 @@ func (b Buf) Update(ctx context.Context, modDir string) error {
 	return b.runCommand(ctx, cmd...)
 }
 
+// Migrate runs the buf Migrate command for the files in the app directory.
```

Comment:
What do you think of moving this to `ignite doctor`? This is a one time thing and that would make sense to have that logic there maybe, given that it is short lived.

If you scaffold a new chain you'll get v2 directly anyway, and if you migrate, we always advice to use `ignite doctor`. That would be a great separation of concerns to move that there imho.